db/migrate/20191218214250_add_roles_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddRolesToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :roles, :jsonb
```

Comment:
Should this column have a default?

Reply:
It's allowed to be nil since the `needs_roll_refresh?` method is immediately true if so. Users don't have default project roles, so I think this is ok.

Reply:
yeah - i was assuming a default of `{}` vs nil but up to you